src/context-logger.spec.ts

```diff
@@ -178,4 +178,176 @@ describe('ContextLogger', () => {
       );
     });
   });
+
+  describe('log entry structure', () => {
+    it('should group bindings under configured bindings key when enabled', () => {
```

Comment:
```suggestion
    it('should group bindings under default bindings key when enabled', () => {
```
Using `default` instead of `configured` can be more readable in my opinion, what do you think? Same applies for `context` keyword test.

Reply:
Sure, updated